datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/SdxRuntimeUpgradeService.java

```diff
@@ -148,7 +148,8 @@ private SdxUpgradeResponse initSdxUpgrade(String userCrn, List<ImageInfoV4Respon
         boolean skipBackup = request != null && Boolean.TRUE.equals(request.getSkipBackup());
         boolean skipAtlasMetadata = request != null && Boolean.TRUE.equals(request.isSkipAtlasMetadata());
         boolean skipRangerAudits = request != null && Boolean.TRUE.equals(request.isSkipRangerAudits());
-        boolean skipRangerMetadata = request != null && Boolean.TRUE.equals(request.isSkipRangerMetadata());
+        //CB-18319
+        boolean skipRangerMetadata = true;
```

Comment:
This change will skip the backup of both HMS and ranger metadata. 
Looks like the variable name "skipRangerMetadata" made you confused. It should be skipRangerHMSMetadata, instead.